scripts/bundle-image-pickup.sh

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash
+
+###############################################################################
+# Copyright Contributors to the Open Cluster Management project
+###############################################################################
+
+####################
+## COLORS
+####################
+CYAN="\033[0;36m"
+GREEN="\033[0;32m"
+PURPLE="\033[0;35m"
+RED="\033[0;31m"
+YELLOW="\033[0;33m"
+NC="\033[0m"
+
+log_color () {
+  case $1 in
+    cyan)
+      echo -e "${CYAN}$2 ${NC}"$3
+    ;;
+    green)
+      echo -e "${GREEN}$2 ${NC}"$3
+    ;;
+    purple)
+      echo -e "${PURPLE}$2 ${NC}"$3
+    ;;
+    red)
+      echo -e "${RED}$2 ${NC}"$3
+    ;;
+    yellow)
+      echo -e "${YELLOW}$2 ${NC}"$3
+    ;;
+  esac
+}
```

Comment:
**Minor suggestion.** I'd move this to `scripts/log-colors.sh`, then import here with `source ./scripts/log-colors.sh`

Then it's easier to use in other scripts and sync with other projects like our e2e-test

Reply:
Added `log_color` to `scripts/log-colors.sh`. Great suggestion!